extensions/interactions/GraphInput/directives/GraphInput.js

```diff
@@ -29,7 +29,8 @@ oppia.directive('oppiaInteractiveGraphInput', [
     return {
       restrict: 'E',
       scope: {
-        onSubmit: '&'
+        onSubmit: '&',
+        lastAnswer: '&'
```

Comment:
getLastAnswer: '&lastAnswer'

and use $scope.getLastAnswer()

Reply:
Done